### PR TITLE
[RW-7392] Install nextflow in AoU docker images

### PR DESCRIPTION
**Pull request by yonghaoy**
Copied from: https://github.com/DataBiosphere/terra-cli/blob/main/docker/Dockerfile#L26

### CODE REVIEW

File: terra-jupyter-aou/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.0.4 - 2021-11-24T05:51:34.197137
```
Comment (Qi77Qi) [inline terra-jupyter-aou/CHANGELOG.md:1]: you also need to update version [here](https://github.com/DataBiosphere/terra-docker/blob/master/config/conf.json#L146)

Comment (yonghaoy) [reply terra-jupyter-aou/CHANGELOG.md:1]: ah, thanks! I am about to ask you about this

Comment (yonghaoy) [reply terra-jupyter-aou/CHANGELOG.md:1]: Hello @Qi77Qi can you help to check the new error? I do not know what happend this time. Also, is Leo caching automattly happened after this PR merge? When do I expect that happen? Thanks!

Comment (Qi77Qi) [reply terra-jupyter-aou/CHANGELOG.md:1]: there's no error yet....CI is still building.

For caching, another PR needs to happen.. See the diff in 2 `preparexxx` script in https://github.com/DataBiosphere/leonardo/pull/2219/files...do you have access to jenkins?

Comment (yonghaoy) [reply terra-jupyter-aou/CHANGELOG.md:1]: It has an error, and I just return GHA.  Now it failed again. 
I can connect to Broad VPN, so I should have access to jenkins? What I should do for the another PR? Run those two script locally?

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**